Vagrantfile

```diff
@@ -74,11 +74,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # please see the online documentation at vagrantup.com.
 
   # Every Vagrant virtual environment requires a box to build off of.
-  config.vm.box = "precise64"
+  config.vm.box = "ubuntu/precise64"
```

Comment:
Here you reference precise64, however the url below refers to trusty. Shouldn't it be ubuntu/trusty64?